settings.gradle.kts

```diff
@@ -75,7 +75,7 @@ gitRepositories {
         uri.set("https://github.com/hashgraph/hedera-protobufs.git")
         // choose tag or branch of HAPI you would like to test with
         // This version needs to match tha HAPI version below in versionCatalogs
-        tag.set("v0.34.0")
+        tag.set("v0.35.0")
```

Comment:
@lukelee-sl We should also set  it [here](https://github.com/hashgraph/hedera-services/blob/develop/settings.gradle.kts#L92)